tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -247,13 +240,15 @@ message PartitionDescProto {
   required string partitionName = 1;
   repeated PartitionKeyProto partitionKeys = 2;
   optional string path = 3;
-  optional int32 id = 4;
+  optional int32 partition_id = 4;
```

Comment:
PartitionDescProto is used for various catalog API, for examples, output parameter of getPartitionByPartitionName and getPartitionsByTableName and getAllPartitions, input parameter of alterTable and addPartitions. In case of output parameter, partition id always is not null. But in case of input parameter, partition id will be null because CatalogClient can get partition informs with table name and partition name.